### PR TITLE
Add Translate Command

### PR DESCRIPTION
**Pull request by HoobMD**

### CODE REVIEW

File: cmd/translate.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
+// Copyright © 2018 NAME HERE <EMAIL ADDRESS>
```
Comment (thomasdfischer) [inline cmd/translate.go]: replace with MIT license

File: cmd/translate.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+
```
Comment (thomasdfischer) [inline cmd/translate.go]: we need to fix formatting

File: translate/translate.go [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+			}
+		case "xml", "protobuf", "graphql":
+			return fmt.Errorf("'%v' is not implemented yet", outFmt)
+		default:
+			return fmt.Errorf("unrecognized output format '%v'", outFmt)
+		}
+		return nil
+}
+
+func readToMap(r io.Reader) (map[string]interface{}, error) {
```
Comment (thomasdfischer) [inline translate/translate.go]: factor out to a util package (since this is re-used in infer/infer.go)

**Comment by fschr** [comment]
fixes #30

**Comment by thomasdfischer** [comment]
closing since this is replaced by #32

**Pull request closed without merging**